src/backend/joanie/payment/api.py

```diff
@@ -62,12 +62,9 @@ class CreditCardViewSet(
 
     def get_queryset(self):
         """Custom queryset to get user's credit cards"""
-        username = (
-            self.request.auth["username"]
-            if self.request.auth
-            else self.request.user.username
+        return models.CreditCard.objects.get_cards_for_owner(
+            owner_id=self.request.user.id
```

Comment:
You are not using anymore the `username` but the `user_id`. Are you sure about this change ?

Reply:
Furthermore, this syntax was a trick to save a db query when the request already contains user username in auth dict. IMO I'll revert this change.

Reply:
I will keep it simple and just add in the filter queryset the active payment_provider name.

Reply:
The revert suggest by @jbpenrath was about using the `username`, not removing the method in the manager I think. Am I wrong @jbpenrath ?